app/elm/Combat.elm

```diff
@@ -42,49 +64,60 @@ type Msg
 
 
 update : Msg -> Model -> String -> Model
-update msg model user =
+update msg model userId =
```

Comment:
I removed the enemy counter-attack for now, since that would complicate this a lot, and it doesn't necessarily make sense with larger parties.

One side effect of having a list of enemies is that updating the right enemy in the list as the result of the player's attack adds some complexity. When we handle enemy attacks, we'll want to break some of this out into helper functions, and maybe even think about re-working the model slightly to simplify the logic.